cmd/e4client/e4client.go

```diff
@@ -234,24 +234,30 @@ func loadOrCreateClient(name, password string) (e4.Client, error) {
 	var e4Client e4.Client
 
 	savedClientPath := fmt.Sprintf("./%s.json", name)
-	if _, err := os.Stat(savedClientPath); err == nil {
-		e4Client, err = e4.LoadClient(savedClientPath)
-		if err != nil {
+	saveFile, err := os.OpenFile(savedClientPath, os.O_RDWR, 0600)
+	if err != nil {
```

Comment:
The else here looks a little messy, perhaps we can use a switch and a newline after the cases, so:

```go
dstFile, err := os.OpenFile(persistClientPath, os.O_RDWR, 0600)
switch {
case err == nil:
       ...
       return e4c, nil

default:
       ...
       return e4c, nil
}
```